plugins/todo-backend/src/service/TodoReaderService.ts

```diff
@@ -130,3 +137,23 @@ export class TodoReaderService implements TodoService {
     };
   }
 }
+
+export const todoReaderServiceRef: ServiceRef<TodoService> =
+  createServiceRef<TodoService>({
+    id: 'todo.client',
```

Comment:
```suggestion
export const todoServiceRef: ServiceRef<TodoService> =
  createServiceRef<TodoService>({
    id: 'todo.service',
```

The `reader` word is only part of the concrete class.

Hehe, since this one has `Service` already in its interface name we're in a pickle since it'd look silly with `todoServiceServiceRef`. Yet, it's probably not worth refactoring things just to get around that. So suggesting the above for now :)

Reply:
Changed!